foodsaving/groups/models.py

```diff
@@ -50,8 +57,14 @@ def send_notifications(self):
                 ):
                     p.notify_upcoming_via_slack()
 
-    def add_member(self, user, history_payload=None):
+    # Adds a person to the group marked as being an applicant
+    def add_applicant(self, user):
         GroupMembership.objects.create(group=self, user=user)
+        self.create = History.objects.create(typus=HistoryTypus.GROUP_APPLY, group=self, users=[user, ], )
+
+    # Adds a "full" member to the group, e.g. grants the status of a normal member.
```

Comment:
Thanks for adding documentation. If you add them as docstring, the IDE will help you even more: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings